test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly add test.md in the gitignore file

Reply:
I only added that file for now.

Reply:
@tacettinkustu you can remove it from your repo (but keep it locally) using: `git rm --cached test.md`

learn more here: https://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste